README.md

```diff
@@ -1,2 +1,35 @@
 # 1st-Capstone
-This project is aimed testing the skills of html and css at Microverse
\ No newline at end of file
+This  project is aimed at building a beautiful landing page.
+
+# Built With
+HTML
+CSS
+
+
+# live Demo
+ To see a live demo kindly visit [Capstone](https://amklass11.github.io/1st-Capstone/)
```

Comment:
- [ ] Please move your index.html file from the root to make your live demo work.
- [ ] Please record a video presenting the features of the project.